README.md

```diff
@@ -262,7 +262,7 @@ The native notifications API is hosted at `nativepush-cluster1.pusher.com` and o
 
 You can send pushes by using the `notify` method, either globally or on the instance. The method takes two parameters:
 
-- `interests`: An Array of strings which represents the interests your devices are subscribed to. These are akin to channels in the DDN with less of an epehemeral nature. Note that currently, you can only send to _one_ interest.
+- `interests`: An Array of strings which represents the interests your devices are subscribed to. These are akin to channels in the DDN with less of an epehemeral nature. Note that currently, you can only publish to, at most, _ten_ interests.
```

Comment:
I think it's a question of balance. We don't want to the have the client libs do payload validation because that increases the maintenance burden on us, but on the other hand, it would be nice if we could validate everything in the client, it's a better experience for the customers and marginally more efficient for us. This can be improved by accurate and concise documentation, which we have to do either way. So, that's why the docs specify the upper limit at 10, the lib does not enforce it but the server does.